plugins/plugins.go

```diff
@@ -119,6 +119,17 @@ const (
 	StateWarn State = "WARN"
 )
 
+const (
```

Comment:
For the `State` constants above we've introduced a type alias. How about doing that here, too?

```go
type TriggerMode string
const (
    TriggerPeriodic TriggerMode = "periodic"
     // ... etc
```

Reply:
Done